src/components/RichEditor/index.jsx

```diff
@@ -95,6 +95,7 @@ class RichEditor extends React.Component {
               extended_valid_elements: 'span[lang|id] -span',
               content_css: false,
               content_style: contentStyle,
+              default_link_target: '_blank',
```

Comment:
how does this option solve the issue?

Reply:
This will not resolve the issue because the issue is from the backend, not the frontend
but I think it's a UX bug to open the link in the same tab because it might be a link to another website and it will take the user away from the edX website and it's not a good UX plus I thought the reporter also wants this to be fixed on the frontend

<img width="741" alt="image" src="https://user-images.githubusercontent.com/78806673/211304600-b5f42e54-5d58-499c-9c6f-d7758321ca01.png">